### DISCUSSION TITLE
'For you' tab displays the user's actions (forked, starred) but I'm not following these users

### DISCUSSION THREAD

**Discussion by firassziedan** [community/community#31581] [Other Feature Feedback, Questions, & Ideas]
**The problem**
`For you` tab displays user's actions I never followed. When I opened the dropdown menu beside this, GitHub said `You're seeing this because you follow [user]` and when I open the user profile I'm not following this user.

 
![Screen Shot 2022-09-01 at 11 09 34](https://user-images.githubusercontent.com/18222016/187865847-bd716115-4180-46e9-8389-b5f4a917d2aa.png)

[user profie]
![Screen Shot 2022-09-01 at 11 14 36](https://user-images.githubusercontent.com/18222016/187866137-a782e8e1-d01b-479c-badd-ba5de9bc9d6d.png)

**Comment by mustafa0x** [comment]
same issue. i'm not following these people.

**Comment by ultrabos1429** [comment]
- [ ] English and French 
- [ ] IN 
- [ ] Home 
- [ ] Could 
- [ ] Many
- [ ]  thanks 
- [ ] Ok 
- [ ] R
- [ ] C
- [ ] Bahamas 
- [ 1] Island 
# - [ 1] +971 52 205 3905  ~~~~
# - [ ] +1 86664245999~~~~
# - [ ] +1 24666625999~~~~
## - [ x] []()564655~~~~
@github-staff

**Reply by ultrabos1429** [reply]
Ultrabos1429@gmail.com 
www.botim.com
